.github/workflows/workflow-run-replay-verify.yaml

```diff
@@ -107,7 +107,10 @@ jobs:
 
       - name: Run replay-verify in parallel
         shell: bash
-        run: testsuite/replay_verify.py ${{ matrix.number }} 17 # first argument is the runner number, second argument is the total number of runners
+        run:
+          export PUSH_METRICS_ENDPOINT=http://vmagent-victoria-metrics-agent.victoria-metrics.svc:8429/api/v1/import/prometheus?extra_lable=replay-verify&extra_label=${{ inputs.BUCKET }}-${{ matrix.number }}
```

Comment:
This should work if vmagent is installed on the cluster that this workflow runs on. Probably making an note of that somewhere in the code, otherwise the endpoint is a bit hard to understand since it's k8s internal DNS. We should also make sure that all invocations of `.github/workflows/workflow-run-replay-verify.yaml` run on our self-hosted GHA cluster. 

Also we should have a test plan for this PR before landing